cmd/common-methods.go

```diff
@@ -70,25 +71,59 @@ func parseKey(sseKeys string) (sse string, err *probe.Error) {
 	}
 	decodedString, e := base64.StdEncoding.DecodeString(secretValue)
 	if e != nil || len(decodedString) != 32 {
-		return "", probe.NewError(errors.New("Encryption key should be 32 bytes plain text key or 44 bytes base64 encoded key"))
+		return "", probe.NewError(errors.New("encryption key should be 32 bytes plain text key or 44 bytes base64 encoded key"))
 	}
 	return encryptString[0] + "=" + string(decodedString), nil
 }
 
 // parse and return encryption key pairs per alias.
 func getEncKeys(ctx *cli.Context) (map[string][]prefixSSEPair, *probe.Error) {
+	// SSE-S3
 	sseServer := os.Getenv("MC_ENCRYPT")
+	if sseS3 := os.Getenv("MC_SSE_S3"); sseServer != "" {
+		sseServer = sseS3
+	}
 	if prefix := ctx.String("encrypt"); prefix != "" {
+		fmt.Println("Warning: --encrypt is deprecated, use --sse-s3 instead")
```

Comment:
I think we should use `console.Infof` rather than `fmt.Println`. Remove if it was meant for debugging.

Reply:
👍